diplomacy/persistence/board.py

```diff
@@ -25,6 +25,11 @@ def get_player(self, name: str) -> Player:
         # we ignore capitalization because this is primarily used for user input
         return next((player for player in self.players if player.name.lower() == name.lower()), None)
 
+    def get_players_sorted_by_vscc(self) -> list[Player]:
+        return sorted(
+            self.players, key=lambda sort_player: (len(sort_player.centers) / sort_player.vscc), reverse=True
```

Comment:
This is close to how vscc percent is calculated but not 100% correct

The formula is (currentSC - startingSC) / (VSCC - startingSC)
In other words if you are at 10/25 centers but started at 5, you are 5/20 = 25% there, not 10/25 = 40% there

Reply:
Not sure if we have the startingSC stored anywhere currently though so this may be best for now?

Reply:
We don't, that's also how we sort the scoreboard
I will adress it all at once

Reply:
huh, that's interesting because I copied it from the other place that we computed this to generate the scoreboard. Does this mean the vscc numbers being produces is currently wrong?